com.unity.cinemachine/Runtime/com.unity.cinemachine.asmdef

```diff
@@ -8,7 +8,8 @@
         "Unity.ugui",
         "Unity.RenderPipelines.Universal.Runtime",
         "Unity.2D.PixelPerfect",
-        "Unity.InputSystem"
+        "Unity.InputSystem",
+        "Unity.RenderPipelines.Universal.2D.Runtime"
```

Comment:
Is this a hard dependency or a soft one (other URP dependencies are soft).  What specific part of the code depends on this?

Reply:
So the background on this is that I needed to shift the 2D portion of URP into its own assembly. There were no changes to code that would affect anything about how the pixel perfect camera works. However, because of the shift to the new assembly, now Cinemachine can no longer find the PixelPerfectCamera class without adding this new assembly reference (this is code that exists in CinemachinePixelPerfect.cs). If it was a soft dependency before it still is as I've not changed or added anything. Just moved things around.

Reply:
Actually it looks as though you've added a hard dependency to Unity.RenderPipelines.Universal.2D.Runtime.  It should be be a soft dependency, because CM is supposed to work with or without URP

Reply:
I tested the changes to Cinemachine without URP in my own project, and there were no problems or errors. Seems that all the tests are passing as well.